### DISCUSSION TITLE
porque DataSet.Serialize.Adapter.RESTRequest4D nao funcona com Restrequest4D.

### DISCUSSION THREAD

**Discussion by antoniojprio** [viniciussanchez/dataset-serialize#213] [Q&A] [answered]
porque DataSet.Serialize.Adapter.RESTRequest4D não funciona com Restrequest4D.
uso resp := TRequest.New.BaseURL(BASE_URL)                     
Resource('/login')                    
 .Accept('application/json')
.AddParam('id_usuario', id_usuario.ToString)
.Adapters(TDataSetSerializeAdapter.New(Tablogin))     a esta linha não  encontra a lib.? 
get;

**Comment by viniciussanchez** [comment, marked as answer]
Muito provavelmente você não fez o uso da classe...
https://github.com/viniciussanchez/dataset-serialize-adapter-restrequest4delphi

**Reply by antoniojprio** [reply]
isso mesmo! Muito Obrigado! Parabéns Vinicius pelo Excelente middleware

**Reply by viniciussanchez** [reply]
vlw
